src/assets/img/pswap.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 22 22">
```

Comment:
I guess we don't need it

Reply:
<token-logo /> component will be used instead